src/components/navigation/navigation.scss

```diff
@@ -228,14 +228,19 @@ header {
 .panel-item {
     transition: opacity 0.2s ease;
     width: 100%;
-    padding-right: pxToRem(12);
+    border-radius: pxToRem(6);
+
+    &.active {
+        background-color: rgb(var(--kompendium-contrast-100), 0.5);
+        margin-bottom: pxToRem(20);
```

Comment:
Do we need to add `margin-bottom` when the item is active? I think it is better to remove this change

Reply:
I believe that the extra space below the active item makes it more clear which item within the list is active and where is the next choice to select. It's mostly my "personal taste" in the design I would say. I feel some empty space provides a little extra visual clue for the eyes, when they are searching within the list to find the **currently active** item. Is it OK if we leave it as I intended? 😊 

Do you have a specific reason why it shouldn't be there?

Reply:
The sky blue color clearly shows when the item is selected. I felt a big space between the selected item and not selected one. The other thing I noticed is it doesn't add margin for the last item when selected. It is may be my "personal taste" vs yours 😂, it can stay like that

Reply:
oh I see now in the picture you added 😄 
yes it looks bad for these nested stuff. You're right. I don't want it there neither.
Let me test a bit on my machine